src/UI/Component/Panel/Factory.php

```diff
@@ -95,4 +95,33 @@ public function sub($title,$content);
 	 */
 	public function report($title,$sub_panels);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       Listing Panels are used to list items following all one
```

Comment:
Yes, but it is stronger than "similar type". They should all share some common aspect that defines the nature of the list. E.g. Repository Items, or Appointements. E.g. those two can not be mixed into the same list.